doc/source/tech_note/Land-Only_Mode/CLM50_Tech_Note_Land-Only_Mode.rst

```diff
@@ -1,4 +1,4 @@
-.. _rst_Land-only Mode:
+ _rst_Land-only Mode:
```

Comment:
- [ ] This change results in `undefined label: rst_land-only mode`. Could you please undo? (Or was there a reason you did this?)

Reply:
Good catch.  Not sure how that happened.  Fixed now (but please double-check).